tests/test_utils_url.py

```diff
@@ -28,7 +28,10 @@ def test_url_is_from_any_domain(self):
         self.assertTrue(url_is_from_any_domain(url, ['192.169.0.15:8080']))
         self.assertFalse(url_is_from_any_domain(url, ['192.169.0.15']))
 
-        url = 'javascript:%20document.orderform_2581_1190810811.mode.value=%27add%27;%20javascript:%20document.orderform_2581_1190810811.submit%28%29'
+        url = (
+            'javascript:%20document.orderform_2581_1190810811.mode.value=%27add%27;%20'
+            'javascript:%20document.orderform_2581_1190810811.submit%28%29'
+        )
```

Comment:
This is another example where a strict limit arguably makes an example worse. URL is a single line; splitting them in several lines make it harder (at least for me) to understand the code, and adds a surface for bugs (whitespace errors).